### PR TITLE
(Done mobile) lixi-Create-UI

### PR DESCRIPTION
**Pull request by khanhvinhnguyen**

### CODE REVIEW

**Review by eric-son12 — APPROVED** [review]

File: packages/app-lixi/src/components/Common/EnvelopeCarousel.tsx [hunk 1]
```diff
@@ -12,6 +12,7 @@ export interface EnvelopeCarouselPros {
```
Comment (vince8x) [inline packages/app-lixi/src/components/Common/EnvelopeCarousel.tsx:12]: Please help change the type EnvelopeCarouselPros to EnvelopeCarouselProps

File: packages/app-lixi/src/components/Common/Uploader.tsx [hunk 2]
```diff
@@ -24,17 +25,26 @@ const StyledButton = styled(Button)`
...
   }
 `;
 
 const StyledContainer = styled.div`
-  padding: 10px;
+  padding: 0px;
 `;
 
-export const Uploader = props => {
+export const Uploader = (props: { type: string | Blob; buttonName?: string; buttonType?: string; isIcon: boolean, showUploadList: boolean }) => {
```
Comment (vince8x) [inline packages/app-lixi/src/components/Common/Uploader.tsx:47]: We should define new type UploaderProps if there're too many props fields.
type UploaderProps = {
...
}

File: packages/app-lixi/src/components/Lixi/CreateLixiForm.tsx [hunk 3]
```diff
@@ -451,37 +591,34 @@ const CreateLixiForm = ({ account, disabled }: CreateLixiFormProps) => {
       setActivatedAtLixiIsValid(true);
     }
   };
+
+  const showImage = value => {
+    setNewEnvelopeId(value);
+    const envelope = envelopes.find(item => item.id === value)
+    const src = baseUrl + 'api/' + envelope.thumbnail;
```
Comment (vince8x) [inline packages/app-lixi/src/components/Lixi/CreateLixiForm.tsx:598]: We could use string literal here
`
``${baseUrl}api/${envelope?.thumbnail}``
`

File: packages/lixi-models/src/constants/upload.ts [hunk 4]
```diff
@@ -4,4 +4,13 @@ export const UPLOAD_TYPES = {
   PAGE_AVATAR: 'page_avatar'
 };
 
+export const UPLOAD_BUTTON_TYPE = {
+  primary: 'primary',
```
Comment (vince8x) [inline packages/lixi-models/src/constants/upload.ts:8]: We should use uppercase constant to make it consistent

File: packages/app-lixi/src/components/Lixi/CreateLixiForm.tsx [hunk 5]
```diff
@@ -655,124 +793,290 @@ const CreateLixiForm = ({ account, disabled }: CreateLixiFormProps) => {
...
+            Ok
+          </Button>
+        ]}>
+        <EnvelopeCarousel envelopes={envelopes} handleChangeEnvelope={handleChangeEnvelope} />
+      </Modal >
     </>
-  );
+
+  )
+  //     {/* Lixi envelope */ }
```
Comment (vince8x) [inline packages/app-lixi/src/components/Lixi/CreateLixiForm.tsx:1055]: Please remove unused code

**Pull request merged**